aci/convert/convert.go

```diff
@@ -188,22 +190,30 @@ type projectAciHelper types.Project
 
 func (p projectAciHelper) getAciSecretVolumes() ([]containerinstance.Volume, error) {
 	var secretVolumes []containerinstance.Volume
-	for secretName, filepathToRead := range p.Secrets {
-		data, err := ioutil.ReadFile(filepathToRead.File)
-		if err != nil {
-			return secretVolumes, err
+	for _, svc := range p.Services {
+		secretServiceVolume := containerinstance.Volume{
+			Name:   to.StringPtr(serviceSecretPrefix + svc.Name),
+			Secret: make(map[string]*string),
 		}
-		if len(data) == 0 {
-			continue
+		for _, scr := range svc.Secrets {
+			data, err := ioutil.ReadFile(p.Secrets[scr.Source].File)
+			if err != nil {
+				return secretVolumes, err
+			}
+			if len(data) == 0 {
+				continue
+			}
+			dataStr := base64.StdEncoding.EncodeToString(data)
+			if scr.Target == "" {
+				scr.Target = scr.Source
```

Comment:
This will support onlt renaming the file, but target cannot be a path (i.e. include "/" or "\"). 
Fine for this PR, we should implement support path other than /run/secrets in a separate PR. Tested with @ulyssessouza, if we mount a secret for example with `target=/etc/nginx/conf.d/site.conf` then entire will be wiped by ACI to provide the secrets, secret files cannot be mixed with other files in the same folder